### ISSUE TITLE
can't close the drawer when touchToDrag set to false

### ISSUE THREAD

**Issue by smokingbarrel** [jakiestfu/Snap.js#37]
Unable to close the drawer with following setting:

 touchToDrag: false,

I would like to disable drag feature and be able open/close drawers using just toggle botton(-s) as well as be able to close drawer when clicking on the content area (just like when touchToDrag is enabled)

Totally sick plugin btw, congrats.

**Comment by KevinTCoughlin** [comment]
Hey @smokingbarrel can you post your code? I was able to create a toggle button with my snapper's touchToDrag property set to false and using the following click handler code:

```
        document.getElementById("toggleSnapBtn").addEventListener('click', function(){
            if( snapper.state().state=="left" ){
                snapper.close();
            } else {
                snapper.open('left');
            }
        });
```

**Comment by t0rpedo** [comment]
Thanks @KevinTCoughlin - Snagging that little snippet.

**Comment by jakiestfu** [comment]
@smokingbarrel @KevinTCoughlin @t0rpedo Just so you guys know, I've reclarified toggles a little bit in the new [FAQ](https://github.com/jakiestfu/Snap.js#--how-do-i-make-a-toggle-button) section on the readme.

That being said, @smokingbarrel and his problem still apply, not being able to `tapToClose` with `touchToDrag` disabled. I will look into this

**Comment by smokingbarrel** [comment]
Hey Kevin,
My code is as supplied. The only difference is that I changed touchToDrag variable from true to false. I was assuming that this change would just simply disable the drag, leaving the rest of functionality intact.

I'll wait and see if Jacob will do the fix.

Note for Jacob: I've noticed that in the toggles example, a toggle only opens the drawer but if you click it again it doesn't close it. Would be cool if each toggle was responsible for opening and closing the corresponding drawer

Many thanks
